app/server/utils/transaction_limits.py

```diff
@@ -0,0 +1,119 @@
+
+        if len(kyc) == 0:
+            return False
+
+        if kyc[0].kyc_status == 'VERIFIED' and kyc[0].type == 'INDIVIDUAL':
+            return True
+
+    return False
+
+
+def check_user_kyc_two_id_verified(credit_transfer):
+    if check_user_liquid_token_type(credit_transfer):
+        return False
+
+    if credit_transfer.sender_user is not None:
+        kyc = credit_transfer.sender_user.kyc_applications
+
+        if len(kyc) == 0:
+            return False
+
+        if kyc[0].kyc_status == 'VERIFIED' and kyc[0].type == 'BUSINESS':
+            return True
+
+    return False
+
+
+class TransferLimit(object):
+    def __init__(self, name: str, total_amount: int=None, time_period_days: int=None,
+                 transfer_balance_percentage: float=None, transfer_count: int=None,
+                 applied_to_transfer_types: List=None, application_filter: Callable=None):
+        self.name = name
+        self.total_amount = total_amount
+        self.time_period_days = time_period_days
+        self.transfer_balance_percentage = transfer_balance_percentage
+        self.transfer_count = transfer_count
+        self.applied_to_transfer_types = applied_to_transfer_types
+        self.application_filter = application_filter
+
+
+LIMITS = [
+    TransferLimit('Sempo Level 0', 5000, 7, None, None, ['PAYMENT'], check_user_but_no_phone_verified_and_no_kyc),
+    TransferLimit('Sempo Level 0', 10000, 30, None, None, ['PAYMENT'], check_user_but_no_phone_verified_and_no_kyc),
+    TransferLimit('Sempo Level 0', 0, 30, None, None, ['WITHDRAWAL', 'DEPOSIT'], check_user_but_no_phone_verified_and_no_kyc),
+
+    TransferLimit('Sempo Level 1', 5000, 7, None, None, ['PAYMENT'], check_user_is_phone_verified_but_no_kyc),
+    TransferLimit('Sempo Level 1', 20000, 30, None, None, ['PAYMENT'], check_user_is_phone_verified_but_no_kyc),
+    TransferLimit('Sempo Level 1', 0, 30, None, None, ['WITHDRAWAL', 'DEPOSIT'], check_user_is_phone_verified_but_no_kyc),
+
+    TransferLimit('Sempo Level 2', 50000, 7, None, None, ['PAYMENT'], check_user_kyc_verified),
+    TransferLimit('Sempo Level 2', 100000, 30, None, None, ['PAYMENT'], check_user_kyc_verified),
+    TransferLimit('Sempo Level 2', 50000, 7, None, None, ['WITHDRAWAL', 'DEPOSIT'], check_user_kyc_verified),
+    TransferLimit('Sempo Level 2', 100000, 30, None, None, ['WITHDRAWAL', 'DEPOSIT'], check_user_kyc_verified),
+
+    TransferLimit('Sempo Level 3', 500000, 7, None, None, ['PAYMENT'], check_user_kyc_two_id_verified),
+    TransferLimit('Sempo Level 3', 1000000, 30, None, None, ['PAYMENT'], check_user_kyc_two_id_verified),
+    TransferLimit('Sempo Level 3', 500000, 7, None, None, ['WITHDRAWAL', 'DEPOSIT'], check_user_kyc_two_id_verified),
+    TransferLimit('Sempo Level 3', 1000000, 30, None, None, ['WITHDRAWAL', 'DEPOSIT'], check_user_kyc_two_id_verified),
+
+    TransferLimit('GE Liquid Token - Standard User', None, 7, 0.1, 1, ['WITHDRAWAL'], check_user_liquid_token_type)
```

Comment:
so when we create a GE token we label it as liquid? why this distinction for GE token instead of labeling a GE token as GE?

Reply:
@enjeyw

Reply:
because that's what it's technically called. i think it makes more sense to name it after the thing it is, rather than the client